1-js/02-first-steps/10-ifelse/article.md

```diff
@@ -1,239 +1,244 @@
-4. If that's true -- it returns `'Greetings!'`. Otherwise, it continues to the expression after the last colon '":"', returning `'What an unusual age!'`.
+1. Det første spørsmålstegnet sjekker om `alder < 3`.
+2. Om det er "true" returnerer den `'Hei spedbarn!'`. Hvis den ikke er "true", går den til neste verdien etter kolontegenet '":"', som er `alder < 18`.
+3. Om det "true", returner den `'Hallo!'`. Hvis den ikke er "true", går den til neste verdien etter kolontegenet '":"', som er `age < 100`.
+4. Om det "true", returner den `'God dag!'`. Hvis den ikke er "true", går den til neste verdien etter kolontegenet '":"', som er `'What an unusual age!'`.
 
-Here's how this looks using `if..else`:
+Her kan du se hvordan det ville sett ut med `if..else`:
 
 ```js
-if (age < 3) {
-  message = 'Hi, baby!';
-} else if (age < 18) {
-  message = 'Hello!';
-} else if (age < 100) {
-  message = 'Greetings!';
+if (alder < 3) {
+  melding = "Hei spedbarn!";
+} else if (alder < 18) {
+  melding = "Hallo!";
+} else if (alder < 100) {
+  melding = "God dag!";
 } else {
-  message = 'What an unusual age!';
+  melding = "For en høy alder!";
 }
 ```
 
-## Non-traditional use of '?'
+## Utradisjonell bruk av '?'
 
-Sometimes the question mark `?` is used as a replacement for `if`:
+Det hender at spørsmålstegnet `?` blir brukt som et alternativ for `if`:
 
 ```js run no-beautify
-let company = prompt('Which company created JavaScript?', '');
+let selskap = prompt('Hvilket selskap lagde JavaScript?', '');
 
 *!*
-(company == 'Netscape') ?
-   alert('Right!') : alert('Wrong.');
+(selskap == 'Netscape') ?
+   alert('Stemmer!') : alert('Feil.');
 */!*
 ```
 
-Depending on the condition `company == 'Netscape'`, either the first or the second expression after the `?` gets executed and shows an alert.
+Avhengig om påstanden `selskap == 'Netscape` stemmer eller ikke, vil enten den første eller andre koden etter `?` bli kjørt.
 
-We don't assign a result to a variable here. Instead, we execute different code depending on the condition.
+Her satt vi ikke resultatet til en variabel, men kjørte forskjellig kode avhengig av påstanden.
 
-**We don't recommend using the question mark operator in this way.**
+**Vi anbefaler ikke at man bruker spørsmålstegnet på denne måten.**
 
-The notation is shorter than the equivalent `if` statement, which appeals to some programmers. But it is less readable.
+Å skrive det på denne måten er kortere enn en `if` påstand, som kan være attraktivt for mange programmerere. Men det er vanskeligere å lese, og sees dermed på som dårlig vane.
 
-Here is the same code using `if` for comparison:
+Her kan du sammenligne den samme koden med en `if` påstand:
 
 ```js run no-beautify
-let company = prompt('Which company created JavaScript?', '');
+let selskap = prompt('Hvilket selskap lagde JavaScript?', '');
 
 *!*
 if (company == 'Netscape') {
```

Comment:
Company here should be "selskap", sorry for missing this...

Reply:
It is stated in the contributions guide that only the article is to be translated. Code examples should stay as is. PR looks good otherwise. I am a bit unsure about tasks though.
What do you think @iliakan?

Reply:
Oh okay, i guess that's a valid point. What's the reason as to why the code examples shouldn't be translated as well? However i will do it now!

Reply:
That is probably to keep the code working as expected across all of the different translations in this tutorial. This looks good to me now :) good job!